src/script/standard.cpp

```diff
@@ -642,3 +643,19 @@ std::optional<std::vector<std::tuple<int, CScript, int>>> InferTaprootTree(const
 
     return ret;
 }
+
+std::vector<std::tuple<uint8_t, uint8_t, CScript>> TaprootBuilder::GetTreeTuples() const
+{
+    assert(IsComplete());
+    std::vector<std::tuple<uint8_t, uint8_t, CScript>> tuples;
+    if (m_branch.size()) {
+        const auto& leaves = m_branch[0]->leaves;
+        for (const auto& leaf : leaves) {
+            assert(leaf.merkle_branch.size() < TAPROOT_CONTROL_MAX_NODE_COUNT);
```

Comment:
4542ba12866c5f9e7bb0da186800f2f5e450bea6: `<= TAPROOT_CONTROL_MAX_NODE_COUNT `? See also `TaprootBuilder::Insert`. In `descriptor.cpp` we check `branches.size() > TAPROOT_CONTROL_MAX_NODE_COUNT`.

Reply:
Fixed.